tests/system/test_streams.py

```diff
@@ -74,7 +82,11 @@ def test_stream_stream(echo):
         contents.append(response.content)
     assert contents == ['hello', 'world!']
 
-    assert responses.trailing_metadata() == metadata
+    response_metadata = [
+        (metadata.key, metadata.value)
+        for metadata in responses.trailing_metadata()
+    ]
+    assert metadata[0] in response_metadata
```

Comment:
nit: the local `metadata` in the list comprehension has the same name as the `metadata` global variable in the `assert`. Semantically correct, but I find it confusing. Suggest a rename:

```suggestion
    response_metadata = [
        (datum.key, datum.value)
        for datum in responses.trailing_metadata()
    ]
    assert metadata[0] in response_metadata
```

Or, **better yet**, the global constant should be in all caps (`METADATA`) as [per the style guide](https://google.github.io/styleguide/pyguide.html#316-naming)

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/1999/commits/921f20baf070ea7b6c65add40f089a0674dc7d07